src/pages/Chart/index.js

```diff
@@ -28,9 +28,8 @@ export default graphql(ALL_INSIGHTS_BY_PAGE_QUERY, {
   return (
     <div className={styles.wrapper + ' page'}>
       <Query query={USER_SUBSCRIPTIONS_QUERY}>
-        {({ data }) => {
-          const subscription =
-            data && getCurrentSanbaseSubscription(data.currentUser)
+        {({ data: { currentUser } = {} }) => {
+          const subscription = getCurrentSanbaseSubscription(currentUser)
```

Comment:
sometimes backend doesn't return any data here and app is down. Will be nice to avoid it

Reply:
Yes, it crashed because `data` was `undefined` and at the same time destructuring assignment is taken place. Here is the 3 characters fix: `= {}` - assigning default value for `data`